[2025-07-28T23:21:05.393+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-28T23:21:05.406+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-07-28T23:21:04.451933+00:00 [queued]>
[2025-07-28T23:21:05.411+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-07-28T23:21:04.451933+00:00 [queued]>
[2025-07-28T23:21:05.411+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-28T23:21:05.420+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-07-28 23:21:04.451933+00:00
[2025-07-28T23:21:05.425+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-28T23:21:05.426+0000] {standard_task_runner.py:72} INFO - Started process 97 to run task
[2025-07-28T23:21:05.426+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'manual__2025-07-28T23:21:04.451933+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpd6io1o5_']
[2025-07-28T23:21:05.427+0000] {standard_task_runner.py:105} INFO - Job 16: Subtask validate_environment
[2025-07-28T23:21:05.460+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment manual__2025-07-28T23:21:04.451933+00:00 [running]> on host b2497d67dac2
[2025-07-28T23:21:05.525+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kanewyp' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T23:21:04.451933+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T23:21:04.451933+00:00'
[2025-07-28T23:21:05.526+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-28T23:21:05.553+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-28T23:21:05.554+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env &&\n        echo "Environment validation successful."\n        ']
[2025-07-28T23:21:05.564+0000] {subprocess.py:99} INFO - Output:
[2025-07-28T23:21:05.568+0000] {subprocess.py:106} INFO - Validating environment...
[2025-07-28T23:21:05.568+0000] {subprocess.py:106} INFO - Environment validation successful.
[2025-07-28T23:21:05.568+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-28T23:21:05.589+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-28T23:21:05.590+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=manual__2025-07-28T23:21:04.451933+00:00, execution_date=20250728T232104, start_date=20250728T232105, end_date=20250728T232105
[2025-07-28T23:21:05.642+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-28T23:21:05.665+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-28T23:21:05.666+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
